bundles.config.js

```diff
@@ -157,6 +157,16 @@ exports.jsBundles = {
       includePolyfills: false,
     },
   },
+  'amp.js': {
+    srcDir: './src/',
+    srcFilename: 'amp.js',
+    destDir: './dist',
+    minifiedDestDir: './dist',
+    options: {
+      minifiedName: 'v0.js',
+      includePolyfills: true,
+    },
+  },
```

Comment:
The reason I didn't originally add `amp.js` to this list was because it uses additional command line args that are visible to the main `gulp` process but not to the `server.js` process that does the lazy building.

https://github.com/ampproject/amphtml/blob/b009ab7277045b9b3e5b38576dd61301522f59ab/build-system/tasks/helpers.js#L176-L194

Do we not care about options like `wrapper` and `esmPassCompilation` while lazy building?

Reply:
In addition, we can remove `compileCoreRuntime ` and add another promise to this function:

https://github.com/ampproject/amphtml/blob/6afb5c0e47bd8a2ad0df902940b7bb52d4d97e07/build-system/tasks/helpers.js#L196-L219

... that does:

```js
    doBuildJs(jsBundles, 'amp.js', {
      watch,
      minify,
      wrapper: wrappers.mainBinary,
      singlePassCompilation: argv.single_pass,
      esmPassCompilation: argv.esm,
      includeOnlyESMLevelPolyfills: argv.esm,
    }), 
```